### PR TITLE
Fix: HBO Max not showing titles

### LINKED ISSUE #3428: [Presence | Bug] HBO Max
**Describe the Bug**

**Steps to Reproduce**

1. Add the HBO Max presence from the premid store
2. Login into HBO Max on the website and click any show or movie
3. View the results on discord

**Expected Behavior**

Before the presence broke, it showed the movie and time left of the movie. 
Now, it just say's "HBO Max - Feature" nothing else.
**Screenshots**

image

**Environment**

- OS:  Windows 10
- Browser:  Microsoft Edge
- Version:  Version 89.0.774.68 (Official build) (64-bit)
  

**Additional context**

Also, if it's possible to add the feature to make the presence show which category we're browsing. 
For example, there's different categories like, Series, Movies, Originals, Just added, Last Chance, Coming soon, Action, Comedy, 
etc. 
image
image.
 The presence just show's "Browsing" when you click on a category - and when we click a movie to view what it's about it, it can say "Browsing Movie - The Avengers (2012)". Also, a feature to show what you're searching up on HBO Max, same like the netflix rich presence from Tim.

### PR DESCRIPTION
**Pull request by thepoliticallycorrect**
This fixes and refactors the whole Presence so that it correctly shows titles for HBO Max content.

Screenshots:

Watching a series:
image

Watching a movie:
image

Browsing:
image

Search:
image

closes #3428

### CODE REVIEW

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Hi, seems great from the offset, but I think you should include more subpages (if there are any), just supports the homepage and vid page currently. 

Amazing to see finally someone has addressed this

**Comment by EncryptedDev** [comment]
Resolves #3428

**Comment by kagbogre** [comment]
> Hi, seems great from the offset, but I think you should include more subpages (if there are any), just supports the homepage and vid page currently.
> 
> Amazing to see finally someone has addressed this

Now supports all the subpages available in the menu:

image

Example:
image

File: websites/H/HBO Max/dist/metadata.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "play.hbomax.com",
     "help.hbomax.com"
   ],
-  "version": "1.0.5",
+  "version": "1.1.0",
```
Comment (EncryptedDev) [inline websites/H/HBO Max/dist/metadata.json:18]: I'd say bump to 2.0.0 as it is quite significant changes 
```suggestion
  "version": "2.0.0",
```

File: websites/H/HBO Max/dist/metadata.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "https://schemas.premid.app/metadata/1.0",
   "author": {
-    "name": "PreMiD",
-    "id": "503557087041683458"
+    "name": "Kedi",
+    "id": "238668511222693888"
```
Comment (EncryptedDev) [inline websites/H/HBO Max/dist/metadata.json:5]: If you're now the author, you should of changed the client ID too

File: websites/H/HBO Max/dist/metadata.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "https://schemas.premid.app/metadata/1.0",
```
Comment (EncryptedDev) [inline websites/H/HBO Max/dist/metadata.json:2]: Not required, but you might as well bump the schema while you're doing these changes `1.4` is latest - https://schemas.premid.app/metadata/1.4

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/H/HBO Max/dist/metadata.json (hunk 2 above)
Comment (kagbogre) [reply websites/H/HBO Max/dist/metadata.json:5]: Created new clientId 879535934977245244

Proof that it works with the new clientId:

image

File: websites/H/HBO Max/dist/metadata.json (hunk 3 above)
Comment (kagbogre) [reply websites/H/HBO Max/dist/metadata.json]: Bumped metadata version

**Review by Slowlife01 — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**